Gruntfile.js

```diff
@@ -60,7 +60,7 @@ module.exports = function (grunt) {
 				requireSpaceBeforeObjectValues: true,
 				validateIndentation: '\t',
 				validateLineBreaks: 'LF',
-				validateQuoteMarks: true,
+				validateQuoteMarks: {mark: '\'', escape: false},
```

Comment:
I changed it to use single quotes, but I figured it would be better to explicitly state the style of quotes used. You do not agree?